Source/Core/Core/BootManager.cpp

```diff
@@ -245,6 +245,15 @@ bool BootCore(const std::string& _rFilename)
 	if (!StartUp.AutoSetup(SConfig::BOOT_DEFAULT))
 		return false;
 
+	// Block running anything other than Melee and homebrew
+	if (!StartUp.GetGameID().empty() && !StartUp.GameHasDefaultGameIni())
+	{
+		PanicAlertT("This does not seem to be a copy of Super Smash Bros. Melee. "
+		            "Please use a non-Slippi version of Dolphin for running games "
```

Comment:
could we just tell them to go to the official dolphin website?

Reply:
Sure, if that sounds fine to you. Want me to add the link to the message? (I don't think I can make the link clickable in a panic alert, but the URL is short enough that it shouldn't be too hard to type it in manually.)

Reply:
yeah a link sounds good